bin/ChakraCore/CMakeLists.txt

```diff
@@ -75,3 +75,6 @@ if(NOT CC_XCODE_PROJECT)
     ${CHAKRACORE_BINARY_DIR}/
     )
 endif()
+
+add_dependencies(ChakraCore ChakraCoreHeaders)
```

Comment:
I am in the process of adding major revisions to the xplat build system. These revisions have three main features...

1. Greatly reduces compile time for swap between Shared and Static Libraries... This means that a minimal number of files get changed each time.
2. A new ChakraCoreStatic target. This is simply the ChakraCore target with the static preprocessor define added.
3. The number of static libraries is reduced by converting the existing library to an object.